### DISCUSSION TITLE
Ubuntu 23.04 unbooting on ARM and Intel

### DISCUSSION THREAD

**Discussion by M-Rick** [utmapp/UTM#5343] [Linux]
I wanted to create a new virtualization VM to run Ubuntu 23.04 on my MacBook Pro M2.

I tried first on an Intel Mac, I download a legacy Ubuntu disk image from the Ubuntu website. The Ubuntu image won't boot. I then tried on my ARM MPB M2 with the same result.

I installed Ubuntu 22.04 without troubles and it works as expected. I used the upgrade feature of Ubuntu to upgrade to 22.10 and 23.04. It works on the Intel Mac, but not on the ARM Mac.

All the updates worked without issues. But at the final reboot of Ubuntu 23.04 on the ARM mac, I get a never ending loop and impossible to boot Ubuntu Lunar Lobster …

**Configuration 1**  
* UTM Version: 4.2.5
* macOS Version: Ventura 13.4
* Mac Chip (Intel, M1, ...): MBA 13" Intel

**Configuration 2**  
* UTM Version: 4.2.5
* macOS Version: Ventura 13.3
* Mac Chip (Intel, M1, ...): MBP 13" M²

[config (Intel).plist.txt](https://github.com/utmapp/UTM/files/11593014/config.Intel.plist.txt)

[config (M2).plist.txt](https://github.com/utmapp/UTM/files/11592986/config.M2.plist.txt)

**Comment by osy** [comment]
If 22.04 works and 23.04 doesn't, it is an issue with Ubuntu. Have you tried reporting it to them?

**Reply by M-Rick** [reply]
No not yet. I had the same issue later trying to install it directly on an Intel Mac. I could only upgrade from 22.04.

**Comment by M-Rick** [comment]
I tried the same ISO with Parallels Desktop and it works out of the box on both Intel and ARM versions. So it doesn't seem to come from Ubuntu.
